components/match2/commons/match_group_input_util.lua

```diff
@@ -1166,10 +1155,8 @@ function MatchGroupInputUtil.standardProcessMatch(match, Parser, mapProps)
 
 	match.stream = Streams.processStreams(match)
 	match.extradata = Parser.getExtraData and Parser.getExtraData(match, games, opponents) or {}
-	local playallData = {
-		playall = Parser.getPlayAll and Parser.getPlayAll(match.playall) or MatchGroupInputUtil.getPlayAll(match.playall)
-	}
-	match.extradata = Table.merge(playallData, match.extradata)
+	match.extradata = Table.merge({playall = tonumber(match.playall)}, match.extradata)
+	match.bestof = tonumber(match.playall) --For display in existing Bestof?
```

Comment:
Hmm I think I pushed the older changed to github instead of the newer one - I removed it anyways, by most match.bestof calculations, they would already use the number of maps anyways, so the match.bestof is already calculated to be same as match.playall.